### DISCUSSION TITLE
Always Rotating Referance Image

### DISCUSSION THREAD

**Discussion by seemedFilm** [jomjol/AI-on-the-edge-device#2566] [Q&A]
Hi folks,

I am very curious how I get rid of this annoying behavior.

I've seen a page where someone, made everything black except the digits, and placed two new markers on it, to avoid the rotating of the reference image.
I've tried the same here too, with: https://imgur.com/a/8Dyn0QY

Unfortunatly It does not solve it, the image still rotates and the analog counters are out of the scope -> therefor it digits are not read well. 
I am out of ideas what I should do to get rid of it.
I've made several new reference images, markers andn alligment of the digits...

Not sure if this is similar with: 2564

**Comment by friedpa** [comment]
I think the left marker is not set right. Make the marker smaller or extend the frame around the marker, so that the whole area is framed. Maybe the algo is not reacting right, as the left marker is not explicit.

**Comment by seemedFilm** [comment]
I've made the marker bigger then the label and included a bit from the left border. I'll check the behavior after the change.

**Comment by jurasjo** [comment]
Maybe you should set the Alignment Algorithm to Off in the configuration?
![Zrzut ekranu z 2023-08-12 04-58-31](https://github.com/jomjol/AI-on-the-edge-device/assets/19259405/c9720640-bdb1-46cd-8465-57f7d02050da)
